src/frontend/global.css

```diff
@@ -28,6 +28,16 @@ body {
 h1 {
 	font-size: 3.2em;
 	line-height: 1.1;
+	margin: 0.5em 0;
```

Comment:
if you're using this pr to learn how to create a new game event this file can be ignored, the changes are unrelated to implementing the ChangePlayerNameEvent.